dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -146,23 +146,22 @@ boolean isSystemClass(String className) {
   }
 
   /**
-   * Scans class files in the jar file and returns a {@link SymbolReferenceSet} populated with
-   * symbol references.
-   *
-   * @param jar absolute path to a jar file
+   * Returns a set of symbol references (classes to class symbols, method symbols and field symbols)
+   * found in the class path.
    */
-  SymbolReferenceSet scanSymbolReferencesInJar(Path jar) throws IOException {
-    checkArgument(jar.isAbsolute(), "The input jar file path is not an absolute path");
-    checkArgument(Files.isReadable(jar), "The input jar file path is not readable");
+  ClassToSymbolReferences scanSymbolReferencesInClassPath() throws IOException {
+    ClassToSymbolReferences.Builder builder = new ClassToSymbolReferences.Builder();
 
-    SymbolReferenceSet.Builder symbolTableBuilder = SymbolReferenceSet.builder();
-    for (JavaClass javaClass : listClassesInJar(jar)) {
-      if (!isCompatibleClassFileVersion(javaClass)) {
-        continue;
+    for (Path jar : inputClassPath) {
+      for (JavaClass javaClass : listClassesInJar(jar)) {
+        if (!isCompatibleClassFileVersion(javaClass)) {
+          continue;
+        }
+        builder.addAll(scanSymbolReferencesInClass(jar, javaClass));
```

Comment:
I don't think you need "InClass" in this method name. In general, method names don't need to double up on information that's already in the arguments. In this case, the arguments make it clear what you're scanning.

That said, I wonder if this might be clearer if we turned this inside out. That is, 

scanSymbolReferences(jar) 

and push the for loop into scanSymbolReferences

Reply:
Renamed to `scanSymbolReferences`. It makes sense (avoiding Hungarian notation)

Regarding pushing the for loop into scanSymbolReferences, I tried it but I found the method more complex. So keeping the loop in the caller.